### DISCUSSION TITLE
Link to Template in Documentation

### DISCUSSION THREAD

**Discussion by mOctave** [endless-sky/endless-sky#7085] [General]
I don't think this is important enough to be an issue, but I've found that whenever I try to open the outfit graphics template on the wiki (https://github.com/endless-sky/endless-sky/wiki/CreatingOutfits), I either get a 404 error from Google or the page refuses to even try to open. I expect that this is caused by some of the browser plugins I'm using, but I'm thinking it would be a better idea to have the link connect to https://github.com/EndlessSkyCommunity/endless-sky-assets/blob/master/outfits/z%20template.blend. That file appears to be the outfit template, and it's already on GitHub, so it should be openable to anyone who could open the wiki page.

**Comment by 1010todd** [comment]
iirc there's some change to google a while ago where they might clear out inactive accounts. The original link is probably linked to MZ's google drive and MZ is very inactive for the last few years so it might be that.
So I agree it's probably better to link it to the community maintained repo.

**Reply by Zireael07** [reply]
Another possible culprit is Google stealthily changing default to "only show to selected accounts". But yep, anyway it's better to link to somewhere else that's not a GDrive
